gencode.c

```diff
@@ -9418,7 +9425,7 @@ gen_vlan(compiler_state_t *cstate, bpf_u_int32 vlan_num, int has_vlan_tag)
 		 *
 		 * This requires special treatment.
 		 */
-#if defined(SKF_AD_VLAN_TAG_PRESENT)
+#if defined(SKF_AD_VLAN_TAG_PRESENT) || defined(NPCAP_AD_VLAN_TAG_PRESENT)
```

Comment:
Is it true in this context that so long as `SKF_AD_VLAN_TAG_PRESENT` is defined, it does not matter if `NPCAP_AD_VLAN_TAG_PRESENT` is defined, and so long as `SKF_AD_VLAN_TAG_PRESENT` is not defined, `NPCAP_AD_VLAN_TAG_PRESENT` is certainly not defined? In other words, changing this line seems to be a no-op.

Reply:
I intended this to draw attention to the fact that this block is active in a Npcap build as well, clarifying intent of the preprocessor directive, especially in contrast to other places in this file where `SKF_AD_*` constants are instead guarded with `#if defined(__linux__)`.